File: CS/PS/eg/에라토스테네스의체/에토체.js
const get_prime = (num) => {
  let prime_list = Array(num).fill(true);
  let last = Math.floor(Math.sqrt(num));

  for(let i = 2; i < (last + 1); i++){
      if (prime_list[i]){
        for(let j = i + i; j < num; j += i){
          console.log(j);
          prime_list[j] = false;
        }
    }
  }
  return prime_list.slice(2);
  
}

console.log(get_prime(12));

File: CS/PS/eg/에라토스테네스의체/에토체.py
def get_primes(num):
  prime_list = [True] * num
  last = int(num ** 0.5)
  for i in range(2, last + 1):
    if prime_list[i]:
      for j in range (i*2, num, i):
        prime_list[j] = False

  return [i for i in range(2,num) if prime_list[i] == True]